tests/target/type_alias.rs

```diff
@@ -0,0 +1,7 @@
+
+type PrivateTest<'a, I> = (Box<Parser<Input = I, Output = char> + 'a>,
+                           Box<Parser<Input = I, Output = char> + 'a>);
+
+pub type PublicTest<'a, I, O> = Result<Vec<MyLongType>,
+                                       Box<Parser<Input = I, Output = char> + 'a>,
+                                       Box<Parser<Input = I, Output = char> + 'a>>;
```

Comment:
I would like to see some more tests - in particular tests with comments inside the generics and where long type names mean the generics splits over multiple lines. Also a test where the result fits in exactly 100 chars to ensure it ends up on one line.